packages/table/src/index.ts

```
@@ -0,0 +1,799 @@
+                wordWrap: cell.wordWrap ?? this.options.wordWrap,
+            };
+        }
+
+        return {
+            content: String(cell),
+            wordWrap: this.options.wordWrap,
+        };
+    }
+
+    private getLastColor(string_: string): string {
+        const matches = string_.match(/\u001B\[[0-9;]*m/g);
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)